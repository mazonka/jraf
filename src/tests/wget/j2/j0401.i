command=jw md 0 /aaa/ : save 0 /aaa/bbb 10 MTIzNDU2Nzg5MA== : mv 0 /aaa/bbb /aaa/ccc : read 0 /aaa/ccc : get 0 /aaa/bbb