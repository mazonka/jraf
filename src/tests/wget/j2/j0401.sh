#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
rm -rf $wr



