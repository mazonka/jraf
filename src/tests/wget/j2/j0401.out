OK /aaa : OK 10 : OK /aaa/ccc : OK 1 10 MTIzNDU2Nzg5MA== : JRAF_ERR bad path /aaa/bbb